ansible/playbooks/test-kubelet-service-stop/test-kubelet-service-stop.yml

```diff
@@ -0,0 +1,209 @@
+# Description: Verify the behaviour of OpenEBS volumes when kubelet service is stopped on one of the nodes.
+# Author: Swarna
+
+########################################################################################
+# Test Steps:
+#1.Check the maya-apiserver is running.
+#2.Download and Copy Test artifacts to kubemaster.
+#3.Replace PVC name with test case name in percona yaml
+#3.Deploy Percona application with liveness probe running db queries continuously
+#4.Get the Replica name and node name where the replica is scheduled
+#5.Stop the kubelet service on the node where replica is running
+#6.Check the node state after stopping kubectl service
+#7.Check the percona pod status.
+#8.Start the kubelet service on node and check percona pod and replicas are up and running.
+#8.perform Cleanup.
+##########################################################################################
+
+- hosts: localhost
+
+  vars_files:
+    - test-kubelet-service-stop-vars.yml
+
+  tasks:
+
+   - block:
+
+       - include: pre-requisites.yml
+
+
+       - name: Check status of maya-apiserver
+         include_tasks: "{{ansible_env.HOME}}/{{utils_path}}/deploy_check.yml"
+         vars:
+           ns: openebs
+           app: maya-apiserver
+
+       - name: Get percona spec and liveness scripts
+         get_url:
+           url: "{{ item }}"
+           dest: "{{ result_kube_home.stdout }}"
+         delegate_to: "{{ groups['kubernetes-kubemasters'].0 }}"
+         with_items: "{{ percona_links }}"
+
+       - name: Replace volume-claim name with test parameters
+         include_tasks: "{{ansible_env.HOME}}/{{utils_path}}/regex_task.yml"
+         vars:
+           path: "{{ result_kube_home.stdout }}/percona.yaml"
+           regex1: "{{replace_item}}"
+           regex2: "{{replace_with}}"
+         delegate_to: "{{ groups['kubernetes-kubemasters'].0 }}"
+
+
+       - name: Create namespace to deploy application
+         shell: source ~/.profile; kubectl create ns {{ namespace }}
+         args:
+           executable: /bin/bash
+         delegate_to: "{{ groups['kubernetes-kubemasters'].0 }}"
+
+       - name: Create a configmap with the liveness sql script
+         shell: source ~/.profile; kubectl create configmap sqltest --from-file={{ percona_files.1 }} -n {{ namespace }}
+         args:
+           executable: /bin/bash
+         delegate_to: "{{groups['kubernetes-kubemasters'].0}}"
+         register: result
+         failed_when: "'configmap' and 'created' not in result.stdout"
+
+       - include_tasks: "{{ansible_env.HOME}}/{{utils_path}}/deploy_task.yml"
+         vars:
+           app_yml: "{{ percona_files.0 }}"
+           ns: "{{ namespace }}"
```

Comment:
@Swarnalatha-K With latest changes done as per cloud CI requirements to deploy_task/delete_deploy. This util does not work for links, can you make sure to have download application yaml?